server/knexfile.js

```diff
@@ -1,7 +1,7 @@
 // Update with your config settings.
 module.exports = {
   development: {
-    client: "postgresql",
+    client: "pg",
```

Comment:
how come we had to change this? is there any difference?

Reply:
`pg` is what's in knex documentation. I guess one is an alias.